configs/config.development.json

```diff
@@ -0,0 +1,26 @@
+{
+    "socket": 8080,
+
+    "devAssetServer": true,
+
+    "cluster": {
+        "workers": 1,
+        "timeout": 1
+    },
+
+    "hosts": {
+        "static": "/"
+    },
+
+    "i18n": {
+        "langs": ["en", "ru"]
+    },
+
+    "supportedBrowsers": [
```

Comment:
`i18n` and `supportedBrowsers` are used in `.enb/make.js`. They're identical for all environments. Lets' move it in a separate config?